### DISCUSSION TITLE
Any Idea how to get assistant working

### DISCUSSION THREAD

**Discussion by sean-david-welch** [zed-industries/zed#7613] [Feature Requests]
<img width="646" alt="Screenshot 2024-02-09 at 14 02 47" src="https://github.com/zed-industries/zed/assets/105278514/2549f0f7-6953-46a3-a083-07a6ca36de4f">
Any idea how to get assistant working, Ive pasted in my api keys but whenever I ask a question, it does nothing back?

**Comment by mrnugget** [comment]
Do you have credits in your OpenAI platform account? You need to load that up separately from any ChatGPT subscriptions you might have.

**Reply by sean-david-welch** [reply]
Hi Thorsten,  that seems to have done the trick. Thanks for your help!
